2023/AzureSingapore/data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "nileshgule"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Azure Singapore",
+    "Country": "Singapore",
+    "Url": "https://www.meetup.com/en-AU/mssgug/events/291424203",
+    "SessionizeApiId": "{Your sessionize event Id}",
```

Comment:
ApiId is missing, could you please provide this value?

Reply:
@OlenaBorzenko @noopman API ID has been updated. please review the latest changes.

Reply:
Sorry @NileshGule, but I do not see your latest changes here in the PR. I can only see the text "{Your sessionize event Id}" for the API ID.

Reply:
The reason for us being picky is to be helpful and ensure your event is represented as intended on Global Azure! @NileshGule

Reply:
When I go to here [https://github.com/globalazure/communities/commit/d71440b0bb6f54b075c40f7411875785c73d5c55](https://github.com/globalazure/communities/commit/d71440b0bb6f54b075c40f7411875785c73d5c55) It says "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."